deps/RediSearch/src/dep/triemap/triemap.c

```diff
@@ -796,6 +796,7 @@ void TrieMap_IterateRange(TrieMap *trie, const char *min, int minlen, bool inclu
   };
   tmctx.buf = array_new(char, TRIE_INITIAL_STRING_LEN);
   TrieMapRangeIterate(trie->root, min, minlen, max, maxlen, &tmctx);
+  array_free(tmctx.buf);
```

Comment:
Did you informed the Search team?

Reply:
This plug was already on their `master` branch; it must have been added since our last addition of the Search source.